lib/init/config-initializer.js

```diff
@@ -445,110 +448,115 @@ function askInstallModules(modules, packageJsonExists) {
     });
 }
 
-/* istanbul ignore next: no need to test inquirer */
+/* istanbul ignore next: no need to test enquirer */
 /**
  * Ask use a few questions on command prompt
  * @returns {Promise} The promise with the result of the prompt
  */
 function promptUser() {
 
-    return inquirer.prompt([
+    return enquirer.prompt([
         {
-            type: "list",
+            type: "select",
             name: "purpose",
             message: "How would you like to use ESLint?",
-            default: "problems",
+            initial: 1,
```

Comment:
The returned number matches the `name` value of _nth_ in the `choices` array.

Reply:
Can you add that as a comment?

Reply:
@siilwyn can you add this as a comment in the file?

Reply:
Yes :+1: 
(Didn't get a notification for your first comment.)